uber/templates/registration/form.html

```diff
@@ -247,24 +247,28 @@
     </div>
 </div>
 
-{% if c.AT_THE_CON and not attendee.is_new %}
+{% if not attendee.is_new %}
+    {% if c.AT_THE_CON %}
+        <div class="form-group">
+            <label class="col-sm-2 control-label">Received Merch</label>
+            <div class="col-sm-6">
+                {% checkbox attendee.got_merch %} Yes
+            </div>
+        </div>
+    {% endif %}
     <div class="form-group">
-        <label class="col-sm-2 control-label">Received Merch</label>
+        <label class="col-sm-2 control-label">Signed Up</label>
         <div class="col-sm-6">
-            {% checkbox attendee.got_merch %} Yes
+            <i>{{ attendee.registered_local }}</i>
         </div>
     </div>
-
     <div class="form-group">
-        <label class="col-sm-2 control-label">Signed Up</label>
+        <label class="col-sm-2 control-label">Checked In</label>
```

Comment:
only minor note here might be to wrap this in c.AT_OR_POST_CON, because before con, this will always be set to 'never' and not ever be useful info.

But..... this way is fine too.